contrib/packaging/docker/Dockerfile.runtime

```diff
@@ -61,15 +62,9 @@ make -j `getconf _NPROCESSORS_ONLN` && \
 strip bpftool && \
 cd ../../../../ && \
 #
-# bpf-map
-#
-curl -SsL https://github.com/cilium/bpf-map/releases/download/v1.0/bpf-map -o bpf-map && \
-chmod +x bpf-map && \
-strip bpf-map && \
-#
 # cni/loopback
```

Comment:
I think @borkmann already removed this in master.

Reply:
I checked the master branch again and the bpf-map related code still exists. “removed” mean that bpf-map is no longer required to be installed in Dockerfile.runtime?

Reply:
The PR is still in progress as it was blocked by another one to be merged first: https://github.com/cilium/cilium/pull/10542